x-pack/plugins/monitoring/public/plugin.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import './views/all';
+import { i18n } from '@kbn/i18n';
+import {
+  App,
+  AppMountParameters,
+  CoreSetup,
+  CoreStart,
+  Plugin,
+  PluginInitializerContext,
+} from 'kibana/public';
+import {
+  FeatureCatalogueCategory,
+  HomePublicPluginSetup,
+} from '../../../../src/plugins/home/public';
+import { initAngularBootstrap } from '../../../../src/plugins/kibana_legacy/public';
+import { DEFAULT_APP_CATEGORIES } from '../../../../src/core/utils';
+import { MonitoringPluginDependencies, MonitoringConfig } from './types';
+import {
+  MONITORING_CONFIG_ALERTING_EMAIL_ADDRESS,
+  KIBANA_ALERTING_ENABLED,
+} from '../common/constants';
+
+export class MonitoringPlugin
+  implements Plugin<void, void, MonitoringPluginDependencies, MonitoringPluginDependencies> {
+  constructor(private initializerContext: PluginInitializerContext<MonitoringConfig>) {}
+
+  public setup(
+    core: CoreSetup<MonitoringPluginDependencies>,
+    plugins: object & { home?: HomePublicPluginSetup; cloud?: { isCloudEnabled: boolean } }
+  ) {
+    const { home } = plugins;
+    const id = 'monitoring';
+    const icon = 'monitoringApp';
+    const path = '/app/monitoring';
+    const title = i18n.translate('xpack.monitoring.stackMonitoringTitle', {
+      defaultMessage: 'Stack Monitoring',
+    });
+    const monitoring = this.initializerContext.config.get();
+
+    if (!monitoring.ui.enabled || !monitoring.enabled) {
+      return;
+    }
+
+    if (home) {
+      home.featureCatalogue.register({
+        id,
+        title,
+        icon,
+        path,
+        showOnHomePage: true,
+        category: FeatureCatalogueCategory.ADMIN,
+        description: i18n.translate('xpack.monitoring.monitoringDescription', {
+          defaultMessage: 'Track the real-time health and performance of your Elastic Stack.',
+        }),
+      });
+    }
+
+    initAngularBootstrap();
```

Comment:
Does this need to be called here or could it be loaded and called async in the application mount?

Reply:
I actually removed it all together